docs/core/tools/dotnet-nuget-delete.md

```diff
@@ -52,15 +52,15 @@ The `dotnet nuget delete` command deletes or unlists a package from the server.
 
 * **`--no-service-endpoint`**
 
-  Doesn't append "api/v2/package" to the source URL.
+  By default, the command appends "/api/v2/package" to the URL specified by the `--source` option, and that is the URL that it uses to push or delete a package. This option prevents the URL from being changed. The option is for custom feeds that use the source URL exactly as it's specified with the `--source` option. For more information, see the `--source` option later in this article.
 
 * **`--non-interactive`**
 
   Doesn't prompt for user input or confirmations.
 
 * **`-s|--source <SOURCE>`**
 
-  Specifies the server URL. Supported URLs for nuget.org include `https://www.nuget.org`, `https://www.nuget.org/api/v3`, and `https://www.nuget.org/api/v2/package`. For private feeds, replace the host name (for example, `%hostname%/api/v3`).
+  Specifies the server URL. The URL specified by using this option can be either V2 (`https://www.nuget.org/api/v2/`) or V3 (`https://api.nuget.org/v3/index.json`), but the actual URL used to push or delete a package is always `https://api.nuget.org/api/v2/package/PackageId/1.0.0`. For private feeds, replace the host name (for example, `%hostname%/api/v2`).
```

Comment:
```suggestion
  Specifies the server URL. The URL specified by using this option can be either V2 (`https://www.nuget.org/api/v2/`) or V3 (`https://api.nuget.org/v3/index.json`). For private feeds, replace the host name (for example, `%hostname%/api/v3/index.json`).
```

Firstly, the exact URL for packages to be pushed or deleted in nuget.org's v3 feed is theoretically subject to change. The service index (index.json) contains the URL, and therefore should be considered an internal implementation detail, not something to be documented. At least, that's my opinion as someone on the client team. We can get feedback from the server team if we don't agree. Though I agree that it's probably very unlikely to change. Still the point of the service index is that the service index itself is the only URL that needs to stay stable. I also don't think that api.nuget.org is the correct hostname for push, since the service index uses www.nuget.org. But as I mentioned in the other comment, I don't understand how push to v3 feeds can possibly work if it doesn't look up the URL from the service index.

Secondly, we strongly encourage server implementors to implement the V3 protocol, not V2, and customers to use private feeds that implement the V3 protocol. Therefore, I think it's better to use a more realistic V3 endpoint example.